certbot/certbot/_internal/cert_manager.py

```diff
@@ -90,9 +90,23 @@ def certificates(config):
 def delete(config):
     """Delete Certbot files associated with a certificate lineage."""
     certnames = get_certnames(config, "delete", allow_multiple=True)
+    disp = zope.component.getUtility(interfaces.IDisplay)
+    if len(certnames) > 1:
+        suffix = "s"
+        verb = "are"
+    else:
+        suffix = ""
+        verb = "is"
+    logger.info("\nThe following certificate%s %s selected for deletion:\n",
+        suffix, verb)
+    for certname in certnames:
+        logger.info("  * %s", certname)
```

Comment:
Could you please join all of this output into the `yesno` prompt, instead of using `logger`?  

[Here](https://github.com/certbot/certbot/blob/ac8798e8185699cd2b7ddc904bccd978968e8e2f/certbot/certbot/_internal/auth_handler.py#L444-L469) is an example.